### DISCUSSION TITLE
Any Reason for Using jQuery?

### DISCUSSION THREAD

**Discussion by jasondavis** [amitmerchant1990/notepad#37] [Q&A] [answered]
Just curious as this is the first newer project I have seen ion a long time that is using jQuery so I was really curious if there was a reason for it over not using it?  I noticed a lot of other functions that are here that people used to use jQuery for so at a quick glance it looks like jQuery is not used for that much.  Just caught my attention and curiosity

**Comment by amitmerchant1990** [comment, marked as answer]
No concrete reason in particular. I don't want to do a whole refactor just to get rid of jQuery. I don't see it as a trade-off.
Maybe in the future, I'll think about it but not at the moment.
